src/sentry/web/api.py

```diff
@@ -465,6 +466,17 @@ def post(self, request, project, auth, helper, **kwargs):
         if not is_valid_origin(origin, project):
             raise APIForbidden('Invalid document-uri')
 
+        # An invalid CSP report must go against quota
+        if not is_valid_csp_report(report, project):
+            app.tsdb.incr_multi([
+                (app.tsdb.models.project_total_received, project.id),
+                (app.tsdb.models.project_total_blacklisted, project.id),
+                (app.tsdb.models.organization_total_received, project.organization_id),
+                (app.tsdb.models.organization_total_blacklisted, project.organization_id),
+            ])
+            metrics.incr('events.blacklisted')
```

Comment:
@dcramer I think it's ok to lump these under `events.blacklisted`? I'm not sure where else we'd track and I don't want to track against a different tsb key. Likely not worth it.